circle.yml

```diff
@@ -242,6 +242,17 @@ commands:
           name: Generate documentation
           command: make javadoc-dokka
 
+  track-performance:
+    steps:
+      - run:
+          name: Track performance of the Navigation 1.0
+          command: |
+            if [ -n "${MOBILE_METRICS_TOKEN}" ]; then
+              if [[ $CIRCLE_BRANCH == master ]]; then
+                curl -X POST --header "Content-Type: application/json" -d '{"build_parameters": {"CIRCLE_JOB": "android-navigation-benchmark", "BENCHMARK_COMMIT": "${CIRCLE_SHA1}"}}' https://circleci.com/api/v1.1/project/github/mapbox/mobile-metrics/tree/pg-revive-navigation-metrics?circle-token=$MOBILE_METRICS_TOKEN
```

Comment:
> `https://circleci.com/api/v1.1/project/github/mapbox/mobile-metrics/tree/pg-revive-navigation-metrics`

you are triggering a CI run on `pg-revive-navigation-metrics`. This should be `master`?